webpack.config.js

```diff
@@ -27,7 +27,7 @@ module.exports = {
   entry: buildEntries(),
 
   output: {
-    filename: 'bundle.js',
+    filename: '[name].js',
```

Comment:
This is so we can have as many examples as we want, without having to manually name each entry.